tests/models/test_main.py

```diff
@@ -20,6 +20,22 @@
 from tests.helpers import FixtureMixin
 
 
+class TestJSONFieldsAreMutable(BaseApplicationTest):
+
+    def test_json_fields_are_mutable_and_updated(self):
+        test_object = Lot(slug='test-lot', name='Test Lot', data={'test': 'data'})
+        db.session.add(test_object)
+        db.session.commit()
+
+        assert Lot.query.get(test_object.id).data == {'test': 'data'}
+
+        test_object.data['test'] = 'update'
+        db.session.add(test_object)
+        db.session.commit()
+
+        assert Lot.query.get(test_object.id).data == {'test': 'update'}
```

Comment:
For my peace of mind, can we try it with nested data as well?

Reply:
Works now!

Reply:
Hmmmm 🤔

Reply:
Works....... now!

Reply:
OK this doesn't work at all.